web/client/resources/js/goconvey.js

```diff
@@ -769,7 +798,9 @@ function process(data, status, jqxhr)
 	convey.intervalFuncs.momentjs();
 
 	// Show notification, if enabled
-	if (notif())
+	var levelRegex = new RegExp("("+convey.notificationLevel+")", "i");
+	var levelEnabled = current().overall.status.class.match(levelRegex);
```

Comment:
Yeah I think a hash might be a decent compromise. Regexs are a bit of an orbital nuke here I think :D